### PR TITLE
Added arguments to functions

### PR DESCRIPTION
**Pull request by dmtrpro**
Added functionality to get custom headers, footers and sidebars.

### CODE REVIEW

File: Twig/Extension/ThemeExtension.php [hunk 1]
```diff
@@ -45,9 +45,9 @@ public function getFunctions()
     /**
      * Displays Wordpress theme header (and administration menu bar if available).
      */
```
Comment (eko) [inline Twig/Extension/ThemeExtension.php]: Thank you, could you also update PHPdoc on methods your updated please? Just to add the new argument:

```
      * @param string|null $name
```

**Comment by dmtrpro** [comment]
Just added :)

File: Twig/Extension/ThemeExtension.php [hunk 2]
```diff
@@ -44,33 +44,42 @@ public function getFunctions()
...
+    public function getFooter($name = null)
     {
-        \get_footer();
+        \get_footer($name);
     }
 
     /**
      * Displays a Wordpress theme template part.
+     *
+     * @param string $name
```
Comment (eko) [inline Twig/Extension/ThemeExtension.php]: It should be $slug here please ;)

Comment (dmtrpro) [reply Twig/Extension/ThemeExtension.php]: Wow! Sorry) This was a 2 a.m. when I write this)

**Comment by eko** [comment]
Thank you @dmtrpro :)

**Pull request merged**